# Type: fix
# Subtype: I3
# English Summary: Modified the condition judgment in the if statement
# Chinese Summary: 修改 if 语句中的条件判断
# Entry Number: 21

# === Git Diff Content ===

diff --git a/src/gwt/src/org/rstudio/studio/client/workbench/views/console/shell/assist/RCompletionManager.java b/src/gwt/src/org/rstudio/studio/client/workbench/views/console/shell/assist/RCompletionManager.java
index ad0722905..9ec56486c 100644
--- a/src/gwt/src/org/rstudio/studio/client/workbench/views/console/shell/assist/RCompletionManager.java
+++ b/src/gwt/src/org/rstudio/studio/client/workbench/views/console/shell/assist/RCompletionManager.java
@@ -381,7 +381,7 @@ public class RCompletionManager implements CompletionManager
                }
             });
          }
-         else if (input_.hasSelection())
+         else if (!input_.isSelectionCollapsed())
          {
             switch(c)
             {
